File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.py
import sys
input = sys.stdin.readline

cost = [25, 10, 5, 1]
result = [0] * 4
T = int(input())
for _ in range(T):
    C = int(input())
    for i in range(4):
        result[i] = C // cost[i]
        C = C % cost[i]
    print(*result)

File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.js
let fs = require('fs')
let input = fs.readFileSync("/dev/stdin").toString().trim().split("\n")
// let input = fs.readFileSync("./EX.txt").toString().trim().split("\n")

input = input.map(x => parseInt(x))
cost = [25, 10, 5, 1]
result = [0, 0, 0, 0]
input.forEach((x, idx) => {
    if (idx !== 0) {
        for (let i = 0; i < 4; i++) {
            result[i] = Math.trunc(x / cost[i])
            x %= cost[i]
        }
        console.log(...result)
    }
   
})

